Utils/Dataflow/055_documents2TTL/documents2ttl.py

```diff
@@ -150,12 +176,17 @@ def document_glance(data, doc_iri, glance_attrs):
     return ttl
 
 
-def documents_links(data):
-    """
-    Convert documents links to TTL
-    :param data: metadata fro JSON file or stream
-    :return ttl: ttl string with links
-    PAPER atlas:isBasedOn SUPPORTING_DOCUMENT .
+def document_links(data):
+    """ Construct TTL sentences to link paper to its supporting documents.
+
+    The result looks as following:
+    PAPER atlas:isBasedOn SUPPORTING_DOCUMENT
```

Comment:
For now these two lines are rendered into a plain single line (in PDF, at least):

    The result looks as following: PAPER atlas:isBasedOn SUPPORTING_DOCUMENT

The "code" part should at least be mono-typed:

    The result looks as following:
    ``PAPER atlas:isBasedOn SUPPORTING_DOCUMENT``

Maybe started on a new line (with extra empty line between these two), or even turned into a code-block:

    The result looks as following:

    .. code-block:: plain

       PAPER atlas:isBasedOn SUPPORTING_DOCUMENT

Reply:
True, first suggestion implemented in 71305fc.